federatedscope/gfl/trainer/mamltrainer.py

```diff
@@ -1,48 +1,68 @@
 from federatedscope.register import register_trainer
 from federatedscope.core.trainers.trainer import GeneralTorchTrainer
 
+import learn2learn as l2l
 
 class GraphMAMLTrainer(GeneralTorchTrainer):
+    def _hook_on_fit_start_init(self, ctx):
+        super()._hook_on_fit_start_init(ctx)
+        maml = l2l.algorithms.MAML(self.ctx.model, lr=self.cfg.maml.inner_lr)
+        ctx.clone = maml.clone()
+
     def _hook_on_batch_forward(self, ctx):
-        # sub task
         batch = ctx.data_batch.to(ctx.device)
         if self.cfg.model.task.endswith('Regression'):
```

Comment:
This task-dependent condition statement is useful and we may add it into the GeneralTorchTrainer later